xml/signals/DB-HV-1969/appearance-shunting_dwarf.xml

```diff
@@ -54,7 +54,7 @@
 
   <aspecttable>DB-HV-1969</aspecttable>
   <name>Shunting signal, dwarf-type</name>
-  <reference>See <a href="http://www.sh1.org/eisenbahn/shhp.htm">sh1's page</a> or the <a href="http://en.wikipedia.org/wiki/German_railway_signalling#Home_signal.2Fdistant_signal_.28H.2FV.29">Wikipedia</a></reference>
+  <reference>See &lt;a href="http://www.sh1.org/eisenbahn/shhp.htm"&gt;sh1's page&lt;/a&gt; or the &lt;a href="http://en.wikipedia.org/wiki/German_railway_signalling#Home_signal.2Fdistant_signal_.28H.2FV.29"&gt;Wikipedia&lt;/a&gt;</reference>
```

Comment:
See question above about whether this will display OK.

Reply:
Fixed.